### DISCUSSION TITLE
Knife

### DISCUSSION THREAD

**Discussion by gameinside** [SmileYzn/PugMod-AMXX#27] [Ideas]
Hi, it would be nice to add a knife round for choosing a side

**Comment by SmileYzn** [comment]
Already have it. 
Read wiki

**Reply by gameinside** [reply]
kinfe?

**Comment by SmileYzn** [comment]
Yes lol

**Comment by Haco** [comment]
Knife round is available as option in the vote menu (where you can also choose Random, Captains or Balanced teams). But there is no knife round after e. g. captains/leader chose their players (same for the "random" and "balanced" option). Then the match will start immediately, without a knife round.

Maybe a new cvar where you can generally enable/disable the knife round.

For clarity:

1. 10 players on server
2. Vote starts
3. Captains option voted
4. Captains chose all players
5. Knife round 
6. Switch or stay 
7. Live

**Reply by gameinside** [reply]
you can't tell me how to properly configure cvar / Captains selects players. and after that comes the Knife round in pugmod.rc  pls

**Reply by Haco** [reply]
What I described above isn't possible. I've just explained your initial idea to the mod author ( @SmileYzn ). It would be nice if the author could add the feature, so the "knife round" will also be executed (add cvar to enable/disable this feature) after teams are set through captains/by skill balance/random.

1. 10 players on server
2. Vote starts
3. Captains option voted
4. Captains chose all players -> teams are set
5. Knife round       **<-- currently not possible**
6. Knife winner chooses: Switch or Stay?    **<-- currently not possible**
7. Live

**Comment by SmileYzn** [comment]
Why? I can add that without problem lol 😂

I only need to remove Knife round from vote menu, to not get duplicated.

10 player
Vote teams
Knife round
Live

**Reply by SmileYzn** [reply]
I need to go to home, and will start coding this now.

Thanks for tip


Ps. I just added in vote, because in tournament servers the players already enter at correct teams, also I think that can be used with free team changes. 

But people really want to use like captain pick up > knife round way.

Wait for updates 👍

**Reply by Haco** [reply]
Great to hear. Another small feature thats nice2have and adds flexibility:

- Add cvar to hide options in the "Team Vote" menu.

So admins can e. g. only allow "Captains" and "Random" as option in the vote menu.
